src/lib.rs

```diff
@@ -9,10 +9,74 @@
 // except according to those terms.
 
 //! Utilities for random number generation
-//! 
-//! ## Example
-//! 
-//! ```rust
+//!
+//! Rand provides utilities to generate random numbers, to convert them to
+//! useful types and distributions, and some randomness-related algorithms.
+//!
+//! # Basic use
+//!
+//! To quickly get you started, the easiest and most high-level way to just get
+//! some random value is to use [`random()`].
+//!
+//! ```
+//! let x: u8 = rand::random();
+//! println!("{}", x);
+//!
+//! let y = rand::random::<f64>();
+//! println!("{}", y);
+//!
+//! if rand::random() { // generates a boolean
+//!     println!("Heads!");
+//! }
+//! ```
```

Comment:
I'm not a fan of advertising `random`, because it is kind of an antipattern. I don't think it is very useful to tell how to generate "some random value" without saying what kind of random value. I would expect to at least mention that the values are sampled from a uniform distribution and that the float interval is `[0, 1)`.